src/containers/explore/EntityNeighborsContainer.js

```diff
@@ -0,0 +1,177 @@
+  const [visibleOptions, setVisibleOptions] = useState(Map());
+  const [visibleNeighbors, setVisibleNeighbors] = useState(Set());
+  const exploreEntityNeighborsRS :?RequestState = useRequestState([EXPLORE, EXPLORE_ENTITY_NEIGHBORS]);
+
+  const associationEntitySetIds :Set<UUID> = useMemo(() => (
+    neighbors ? neighbors.keySeq().toSet() : Set()
+  ), [neighbors]);
+
+  const dataSetIds :Set<UUID> = useMemo(() => (
+    Set().withMutations((set) => {
+      if (neighbors) {
+        neighbors.reduce((ids :Set<UUID>, esNeighborsMap :Map) => ids.add(esNeighborsMap.keySeq()), set);
+      }
+    }).flatten()
+  ), [neighbors]);
+
+  const dataSetsMap :Map = useSelector(selectOrgDataSets(organizationId, dataSetIds));
+  const associationEntitySetsMap = useSelector(selectOrgDataSets(organizationId, associationEntitySetIds));
+
+  const handleAssociationOnClick = (event :SyntheticEvent<HTMLButtonElement>) => {
+    const { currentTarget } = event;
+    if (currentTarget instanceof HTMLElement) {
+      const { value: dataSetId } = currentTarget;
+      if (dataSetId) {
+        if (visibleOptions.get(dataSetId, Set()).isEmpty()) {
+          const neighborsDataSetIds :Set = neighbors.get(dataSetId, Map()).keySeq().toSet();
+          setVisibleOptions(visibleOptions.set(dataSetId, neighborsDataSetIds));
+        }
+        else {
+          setVisibleOptions(visibleOptions.delete(dataSetId));
+        }
+      }
+    }
+  };
+
+  const handleNeighborOnClick = (event :SyntheticEvent<HTMLButtonElement>) => {
+    const { currentTarget } = event;
+    if (currentTarget instanceof HTMLElement) {
+      const { value: entitySetId } = currentTarget;
+      if (visibleNeighbors.includes(entitySetId)) {
+        setVisibleNeighbors(visibleNeighbors.delete(entitySetId));
+      }
+      else {
+        setVisibleNeighbors(visibleNeighbors.add(entitySetId));
+      }
+    }
+  };
+
+  const associationEntitySetChips = associationEntitySetsMap.valueSeq().map((associationEntitySet :Map) => {
+    const associationEntitySetName = associationEntitySet.get('title') || associationEntitySet.get('name');
+    const associationEntitySetId = associationEntitySet.get('id');
+    return (
+      <Checkbox
+          checked={!visibleOptions.get(associationEntitySetId, Set()).isEmpty()}
+          label={associationEntitySetName}
+          mode="chip"
+          onChange={handleAssociationOnClick}
+          value={associationEntitySetId} />
+    );
+  });
+
+  const neighborDataSetChips = visibleOptions.valueSeq().flatten().toSet().map((neighborESID :UUID) => {
+    const neighborDataSet = dataSetsMap.get(neighborESID, Map());
+    const neighborDataSetName = neighborDataSet.get('title') || neighborDataSet.get('name');
+    const neighborDataSetId = neighborDataSet.get('id');
+    return (
+      <Checkbox
+          checked={visibleNeighbors.includes(neighborDataSetId)}
+          label={neighborDataSetName}
```

Comment:
To fix this: `Warning: Each child in a list should have a unique "key" prop.`

```suggestion
          label={neighborDataSetName}
          key={neighborDataSetId}
```